### ISSUE TITLE
Update UDF contributing docs

### ISSUE THREAD

**Issue by danieldeleo** [GoogleCloudPlatform/bigquery-utils#217]
Update docs to address the issue that was surfaced in [thie PR](https://github.com/GoogleCloudPlatform/bigquery-utils/pull/216#issue-599026822)

**Comment by sdepablos** [comment]
Related to this, on the [UDF documentation](https://github.com/GoogleCloudPlatform/bigquery-utils/tree/master/udfs) there's an explanation on how to add the UDFs to your private project, but the explanation should be improved for the cases the UDF function uses a JS function, as creating the SQL UDF in your project is not enough, you need to upload the JS file to a bucket and change the JS route.

**Comment by danieldeleo** [comment]
Thanks! good catch

**Comment by sdepablos** [comment]
I've created a PR #218 to address it. Let me know if there's something wrong or I can improve it somehow.

**Comment by danieldeleo** [comment]
Thank you for starting the work on this! I need to evaluate the best method of having users locally test with js libs.  It won't be as easy as having them copy js files to their own GCS bucket because we use [webpack](https://webpack.js.org/) to bundle everything into one minified js file. I'm currently testing a method that allows users to run the Cloud Build test pipeline in their own GCP project. Will also evaluate building locally using the [cloud-build-local tool](https://cloud.google.com/build/docs/build-debug-locally#build_locally) but that requires having Docker installed on the local environment.

**Comment by sdepablos** [comment]
Yes, I could only the part about adding the UDF to your private project, not the testing part.
